pandas/core/window.py

```diff
@@ -320,7 +320,52 @@ def aggregate(self, arg, *args, **kwargs):
     agg = aggregate
 
     _shared_docs['sum'] = dedent("""
-    %(name)s sum""")
+    Calculate %(name)s sum of given DataFrame or Series.
+
+    Parameters
+    ----------
+    *args
+        Under Review.
+    **kwargs
+        Under Review.
+
+    Returns
+    -------
+    Series or DataFrame
+        Like-indexed object containing the result of function application
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s  : Calling object with DataFrame data
+    Series.sum  : Equivalent method for Series
```

Comment:
Whitespace seems off on this line and below - are there more than one before the ":"?

Reply:
yup. Deleted.